scripts/installscripts/buster-install-default.sh

```diff
@@ -503,31 +503,22 @@ sudo update-alternatives --install /usr/bin/python python /usr/bin/python3.7 1
 # Install required spotify packages
 if [ $SPOTinstall == "YES" ]
 then
-    sudo apt-get install --yes mopidy=2.2.2-1
-    sudo python2.7 -m pip install Mopidy==2.2.*
-
-    sudo apt-get --yes --allow-downgrades --allow-remove-essential --allow-change-held-packages install libspotify12 python-cffi python-ply python-pycparser python-spotify
-    sudo rm -rf /usr/lib/python2.7/dist-packages/mopidy_spotify*
-    sudo rm -rf /usr/lib/python2.7/dist-packages/Mopidy_Spotify-*
-    cd || exit
-    sudo rm -rf mopidy-spotify
-    git clone -b fix/web_api_playlists --single-branch https://github.com/princemaxwell/mopidy-spotify.git
-    cd mopidy-spotify || exit
-    sudo python2 setup.py install
-    cd || exit
-    # should be removed, if Mopidy-Iris can be installed normally
-    # pylast >= 3.0.0 removed the python2 support
-    sudo pip install pylast==2.4.0
-    # not sure tornado still needs to be downgraded now that Mopidy 3 is not installed and tornado seems to be 5.1
-     sudo pip install 'tornado==5.0'
-    sudo pip install Mopidy-Iris
+    # keep major verson 3 of mopidy
+    echo -e "Package: mopidy\nPin: version 3.*\nPin-Priority: 1001" | sudo tee /etc/apt/preferences.d/mopidy
+
+    wget -q -O - https://apt.mopidy.com/mopidy.gpg | sudo apt-key add -
+    sudo wget -q -O /etc/apt/sources.list.d/mopidy.list https://apt.mopidy.com/buster.list
+    sudo apt-get update
+    sudo apt-get upgrade
```

Comment:
just noticed, this command is missing a `--yes` to avoid asking the user for permission ;-)

```suggestion
    sudo apt-get upgrade --yes
```

Reply:
I've just seen the other scripts got the same issue.